universal-application-tool-0.0.1/app/repository/ApplicantRepository.java

```diff
@@ -5,22 +5,30 @@
 
 import io.ebean.Ebean;
 import io.ebean.EbeanServer;
+import java.util.Comparator;
 import java.util.Optional;
 import java.util.Set;
 import java.util.concurrent.CompletionStage;
 import javax.inject.Inject;
+import models.Account;
 import models.Applicant;
 import play.db.ebean.EbeanConfig;
 
 public class ApplicantRepository {
 
   private final EbeanServer ebeanServer;
   private final DatabaseExecutionContext executionContext;
+  // This is needed in the auth.UatProfileAdapter family of classes, which need to be
+  // initialized before the client objects.  At that time, the ebeanConfig is not available,
+  // so it's impossible to inject this repository into that object.
+  // TODO(someone with guice experience): is there a better way?
+  public static ApplicantRepository INSTANCE;
```

Comment:
Question (blocking): is the issue that EBeanConfig is not available for injection when provideIDCSClient runs?